docs/core/tools/dotnet-nuget-why.md

```diff
@@ -0,0 +1,65 @@
+---
+title: dotnet nuget why command
+description: Shows the dependency graph for a particular package for a given project or solution.
+author: advay26
+ms.date: 05/30/2024
+---
+# dotnet nuget why
+
+**This article applies to:** ✔️ .NET 8.0.4xx SDK and later versions
```

Comment:
@advay26 This version looks wrong, I think

Reply:
@Youssef1313 We have a bug where it's not showing up in `dotnet nuget --help`, because of an incomplete transition to a different CLI parsing library:
* https://github.com/NuGet/Home/issues/13517

but if you run `dotnet nuget why` anyway, you'll see why's help (after error messages talking about missing required arguments)

Otherwise, please explain why you think it's the wrong version.

Reply:
It didn't work for me with .NET 9 SDK Preview 4 but worked with preview 6.

Reply:
.NET 9 P4 came out in May, whereas 8.0.400 and .NET 9 P6 came out in July.

I'll leave it up to the .NET team to decide if they want to update the docs to mention which pre-release version of .NET 9 the feature is available. My opinion is that once .NET 9 goes GA, it won't be relevant any longer and the NuGet team at least doesn't have dedicated docs writer resources to help us keep docs up to date.

Reply:
Thanks for the clarification. I agree the exact preview isn't very relevant.